### ISSUE TITLE
Was Svg2Flutter tool useful?

### ISSUE THREAD

**Issue by maheshwaran-p** [maheshwaran-p/samplesvg#2]

**Comment by anubhavbagri** [comment]
I went to this website and uploaded my svg : https://svg2widget.web.app/

The default code didn't change! Doesn't seem to work!

**Comment by anubhavbagri** [comment]
@maheshwaran-p I'll grateful if you could help!

**Comment by maheshwaran-p** [comment]
Can you please share me your svg ?

**Comment by anubhavbagri** [comment]
Sure, here you go! Thanks in advance..

![emoji](https://user-images.githubusercontent.com/56643117/215815021-a937c043-2f00-4a62-bdef-9f870ac1be1c.svg)

**Comment by anubhavbagri** [comment]
![image](https://user-images.githubusercontent.com/56643117/215815694-0558714a-9e8d-4b04-b019-66e0937be45f.png)

Svg gets uploaded correctly but the code doesn't seem to change; The default candy appears

**Comment by anubhavbagri** [comment]
@maheshwaran-p Any updates? 🥺

**Comment by Kabilan21** [comment]
Hi @anubhavbagri , currently SVG types similar to the one you uploaded are not supported. The attached svg includes an image instead of native svg elements. Hence it is not possible with our tool.

**Comment by Kabilan21** [comment]
@anubhavbagri ,  Please let us know if you were able to convert any svg to widget previously.

**Comment by maheshwaran-p** [comment]
> @maheshwaran-p Any updates? 🥺

yes actually problem in your svg file . you can't use this svg even in flutter_svg @anubhavbagri
